integration/kubernetes/k8s-expose-ip.bats

```diff
@@ -44,7 +44,8 @@ setup() {
 	svcip=$(kubectl get service ${service} -o=json | jq '.spec.clusterIP' | sed 's/"//g')
 	svcport=$(kubectl get service ${service} -o=json | jq '.spec.ports[].port')
 	# And check we can curl the expected response from that IP address
-	curl http://$svcip:$svcport | grep "Hello, world!"
+	echo_msg="hello-world"
+	sudo curl http://$svcip:$svcport/echo?msg=${echo_msg} | grep "$echo_msg"
```

Comment:
- Can we drop `sudo` here?
- I wonder if `echo_msg` should not even use a hyphen as that might get encoded to `%2D`?

Reply:
I really have little knowledge in html format.😭
So I tried a few test, with `echo_msg=hello-world`, `echo_msg=hello,world`, `echo_msg=hello, world`. FWIT, only space character could't be correctly resolved.
$ kubectl get service
NAME         TYPE           CLUSTER-IP      EXTERNAL-IP   PORT(S)          AGE
kubernetes   ClusterIP      10.96.0.1       <none>        443/TCP          4m23s
my-service   LoadBalancer   10.100.65.156   <pending>     8080:32202/TCP   61s
$ curl http://10.100.65.156:8080/echo?msg=hello-world
hello-world
$ curl http://10.100.65.156:8080/echo?msg=hello,world
hello,world
$ curl http://10.100.65.156:8080/echo?msg=hello, world
hello,curl: (6) Could not resolve host: world